src/pages/kyunghoon/Login/Login.js

```diff
@@ -1,8 +1,81 @@
 import React from 'react';
+import './Login.scss';
 
 class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      email: '',
+      password: '',
+    };
+  }
+
+  handleIdInput = event => {
+    this.setState({ email: event.target.value });
+  };
+
+  handlePwInput = event => {
+    this.setState({ password: event.target.value });
+  };
```

Comment:
계산된 속성명을 활용해 다음과 같이 작성할 수 있습니다.
단, 각 input 태그의 name attribute에 해당 state의 이름을 부여해주셔야 합니다.
```suggestion
  handleInput = event => {
    this.setState({ [event.target.name]: event.target.value });
  };
```